### DISCUSSION TITLE
How to make regression fit in a range of [-1, +1]?

### DISCUSSION THREAD

**Discussion by yuxi-liu-wired** [autogluon/autogluon#2547] [Q&A] [answered]
I have a dataset of form (X, Y). Y is a numerical variable known to be in range [-1, +1]. Using arctanh doesn't work, because it often goes exactly to -1, +1.

A histogram of Y looks like
![image](https://user-images.githubusercontent.com/33951560/206722597-31cddc5d-a70a-4b02-9489-f3d3bce36425.png)

**Comment by Innixma** [comment, marked as answer]
If you are using AutoGluon, just post-process the result to be -1, +1 if it is outside the bounds.
